### PR TITLE
Fix upgrade for 9.11.3 which was wrong + fix latest tag + add new latest-postgres tag

### PR DESCRIPTION
**Pull request by vmassol**

### CODE REVIEW

File: library/xwiki [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 GitCommit: bdf1a8464a668061ba564249461633f0445e08a6
 Directory: 9/postgres-tomcat
 
-Tags: 10, 10.1, 10-mysql-tomcat, 10.1-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable
+Tags: 10, 10.1, 10-mysql-tomcat, 10.1-mysql-tomcat, stable-mysql-tomcat, stable-mysql, stable, latest
 GitCommit: ac81dad2ed329469be2ed8b624fd2bc7a44455c4
 Directory: 10/mysql-tomcat
 
-Tags: 10-postgres-tomcat, 10.1-postgres-tomcat, stable-postgres-tomcat, stable-postgres
+Tags: 10-postgres-tomcat, 10.1-postgres-tomcat, stable-postgres-tomcat, stable-postgres, latest-postgres
```
Comment (tianon) [inline library/xwiki]: See https://github.com/docker-library/official-images#tags-and-aliases (and https://github.com/docker-library/official-images/pull/4143 :innocent:). :heart:

**Comment by vmassol** [comment]
@tianon Thanks for the review. I've now fixed it.

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-arches b/_bashbrew-arches
index b1c87d8..e86c199 100644
--- a/_bashbrew-arches
+++ b/_bashbrew-arches
@@ -1,4 +1,4 @@
 xwiki:latest @ amd64
+xwiki:lts @ amd64
 xwiki:lts-postgres @ amd64
-xwiki:stable @ amd64
 xwiki:stable-postgres @ amd64
diff --git a/_bashbrew-list b/_bashbrew-list
index 64b6ed6..92cd40d 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -3,8 +3,8 @@ xwiki:9-mysql-tomcat
 xwiki:9-postgres-tomcat
 xwiki:9.11
 xwiki:9.11-postgres-tomcat
-xwiki:9.11.2
-xwiki:9.11.2-postgres-tomcat
+xwiki:9.11.3
+xwiki:9.11.3-postgres-tomcat
 xwiki:10
 xwiki:10-mysql-tomcat
 xwiki:10-postgres-tomcat
diff --git a/xwiki_latest/Dockerfile b/xwiki_latest/Dockerfile
index df9ab32..5cac9ec 100644
--- a/xwiki_latest/Dockerfile
+++ b/xwiki_latest/Dockerfile
@@ -44,9 +44,9 @@ RUN apt-get update && \
 # Install XWiki as the ROOT webapp context in Tomcat
 # Create the Tomcat temporary directory
 # Configure the XWiki permanent directory
-ENV XWIKI_VERSION=9.11.3
+ENV XWIKI_VERSION=10.1
 ENV XWIKI_URL_PREFIX "http://maven.xwiki.org/releases/org/xwiki/platform/xwiki-platform-distribution-war/${XWIKI_VERSION}"
-ENV XWIKI_DOWNLOAD_SHA256 40dcf3cd92580f4ef411b8d6d7e4454469e37a425408dff49f4a8b24917f2a48
+ENV XWIKI_DOWNLOAD_SHA256 744d5ac88f1976fe5ac2a6a0d1dd9e2e0916d6657e7feb24063b6b657697167a
 RUN rm -rf /usr/local/tomcat/webapps/* && \
   mkdir -p /usr/local/tomcat/temp && \
   mkdir -p /usr/local/xwiki/data && \
diff --git a/xwiki_latest/Dockerfile b/xwiki_lts/Dockerfile
similarity index 100%
copy from xwiki_latest/Dockerfile
copy to xwiki_lts/Dockerfile
diff --git a/xwiki_stable/tomcat/setenv.sh b/xwiki_lts/tomcat/setenv.sh
similarity index 100%
rename from xwiki_stable/tomcat/setenv.sh
rename to xwiki_lts/tomcat/setenv.sh
diff --git a/xwiki_stable/xwiki/docker-entrypoint.sh b/xwiki_lts/xwiki/docker-entrypoint.sh
similarity index 100%
rename from xwiki_stable/xwiki/docker-entrypoint.sh
rename to xwiki_lts/xwiki/docker-entrypoint.sh
diff --git a/xwiki_stable/xwiki/hibernate.cfg.xml b/xwiki_lts/xwiki/hibernate.cfg.xml
similarity index 100%
rename from xwiki_stable/xwiki/hibernate.cfg.xml
rename to xwiki_lts/xwiki/hibernate.cfg.xml
diff --git a/xwiki_stable/Dockerfile b/xwiki_stable/Dockerfile
deleted file mode 100644
index 5cac9ec..0000000
```

</details>

**Comment by yosifkit** [comment]
Build test of #4142; 498b920a; `amd64` (`xwiki`):

```console

$ bashbrew build xwiki:9
Using bashbrew/cache:b44e7fc05c249181ea1004824209c41110f2f71725fa4da88a38bfdd4e6829f9 (xwiki:9)
Tagging xwiki:9
Tagging xwiki:9.11
Tagging xwiki:9.11.3
Tagging xwiki:9-mysql-tomcat
Tagging xwiki:mysql-tomcat
Tagging xwiki:lts-mysql-tomcat
Tagging xwiki:lts-mysql
Tagging xwiki:lts

$ test/run.sh xwiki:9
testing xwiki:9
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:9-postgres-tomcat
Using bashbrew/cache:1246544f6e26020ac73787dfec388053bef7d8fa6dd3b28f782701a1fd614ef6 (xwiki:9-postgres-tomcat)
Tagging xwiki:9-postgres-tomcat
Tagging xwiki:9.11-postgres-tomcat
Tagging xwiki:9.11.3-postgres-tomcat
Tagging xwiki:postgres-tomcat
Tagging xwiki:lts-postgres-tomcat
Tagging xwiki:lts-postgres

$ test/run.sh xwiki:9-postgres-tomcat
testing xwiki:9-postgres-tomcat
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:10
Using bashbrew/cache:c2481e4caa3cbb5b019ead85fb06874bcff21f935aef9ca00b34b7914d85cdea (xwiki:10)
Tagging xwiki:10
Tagging xwiki:10.1
Tagging xwiki:10-mysql-tomcat
Tagging xwiki:10.1-mysql-tomcat
Tagging xwiki:stable-mysql-tomcat
Tagging xwiki:stable-mysql
Tagging xwiki:stable
Tagging xwiki:latest

$ test/run.sh xwiki:10
testing xwiki:10
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build xwiki:10-postgres-tomcat
Using bashbrew/cache:de82e9cd8842c96e07ebfabd2f725d07feb1a53b19deb6326498736503af83f2 (xwiki:10-postgres-tomcat)
Tagging xwiki:10-postgres-tomcat
Tagging xwiki:10.1-postgres-tomcat
Tagging xwiki:stable-postgres-tomcat
Tagging xwiki:stable-postgres

$ test/run.sh xwiki:10-postgres-tomcat
testing xwiki:10-postgres-tomcat
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**